bubblewrap.c

```diff
@@ -1745,6 +1843,13 @@ parse_args_recurse (int          *argcp,
           op->fd = file_fd;
           op->dest = argv[2];
 
+          /* We historically hard-coded the mode of a --file as 0666. */
+          if (next_perms >= 0)
+            op->perms = next_perms;
+          else
+            op->perms = 0666;
```

Comment:
Previously though, since we weren't calling `fchmod()` the `0666` permission was modified by `umask`.  Oh I see, but we invoke `umask(0)` explicitly.

But wait...that means we've been defaulting to making e.g. files *world writable* by default?  That seems...pretty broken?  Obviously not related to your PR, and your PR will let us fix it but...ew?  May be worth adding something to the docs about this.

Reply:
> But wait...that means we've been defaulting to making e.g. files world writable by default?

Yes, for at least `--file`.

>  That seems...pretty broken?

I personally think so, but it's what we always did. If you want to change the defaults, that would be an incompatible change, but maybe a worthwhile one.

> ew?

Agreed. However, note that #265 was going to realign `--file` with `--bind-data` by making `--bind-data` use mode 0666 (which I didn't like).

> May be worth adding something to the docs about this

This branch explicitly documents the defaults in the man page. Previously, you could only find this out by reading the source code or experimenting.

Reply:
Yep agreed with all that!  I should have been clearer there was no problem with your PR, I just found the default surprising.